site/content/en/docs/Examples/simple-genai-gameserver.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Example Build and Run a Simple Gameserver that Connects to an Inference Server"
```

Comment:
Suggest a shorter title - it looks especially long in the sidebar: 

<img width="253" alt="Screenshot 2024-03-15 at 10 45 23 AM" src="https://github.com/googleforgames/agones/assets/7751204/0404989d-c02c-4856-8799-90a31074c54d">

Reply:
Maybe `GenAI GameServer` (it's already nested under Examples, so I don't think we need to put Example in the title).